### PR TITLE
Move I2C ports for more convenient capsule assembly

### PR DESCRIPTION
**Pull request by bbrk24**
The UART ports didn't move, but I did alias `Serial1` to `radioUart`.

### CODE REVIEW

File: sketch_oct9a.ino [hunk 1]
```diff
@@ -99,19 +102,19 @@ int8_t radiansToCappedDegrees(float rad) {
...
 }
 
 #if CAPSULE == 2
-void SERCOM3_Handler(void) {
+void SERCOM1_Handler(void) {
   humidityI2C.onService();
 }
 #endif
+
+void SERCOM3_Handler(void) {
```
Comment (DarthSethious1) [inline sketch_oct9a.ino:115]: why was this moved?

File: sketch_oct9a.ino [hunk 2]
```diff
@@ -231,7 +234,7 @@ void sendDataToRadio() {
     buf,
     sizeof buf,
     "%c%08X,%c%08X,%07X,%X,%c%02X,%c%02X,%c%02X,%c%04X,%c%04X,%c%04X,%c%04X"
-#if CAPSULE = 2
+#if CAPSULE == 2
```
Comment (DarthSethious1) [inline sketch_oct9a.ino:237]: good catch

File: sketch_oct9a.ino (hunk 1 above)
Comment (bbrk24) [reply sketch_oct9a.ino:115]: I reordered them so that they're still in numerical order of the SERCOMs. I suppose I didn't have to.

Comment (DarthSethious1) [reply sketch_oct9a.ino:115]: oh okay, jsut wondering
